### PR TITLE
add storybook for copyright container

### LINKED ISSUE #2195: Add container story for CopyRight Container
**Is your feature request related to a problem? Please describe.**
As we have moved components over to the Psammead component library, we have few items in our Simorgh Storybook: https://bbc.github.io/simorgh/

We should include our container components in the Simorgh storybook.

**Describe the solution you'd like**
Add story to CopyRight Container (on storybook)

- [x] What would be ideal, would be to add a drop-down that could switch between services and populate these with the correct view. e.g. selecting Persian from the drop-down when looking at the CopyRightContainer will show you the News | Persian version.
- [x] The containers relating to RTL services, (e.g. Persian and in future, other RTL services) should have the `dir="rtl"` attribute passed through to Storybook. Could be added via a Decorator, that gets the value of `dir` from the Service Context. 

See the implementation of this for the Psammead sitewide links component - https://bbc.github.io/psammead/?path=/story/components-sitewidelinks--default When you select `Persian` from the `language` dropdown, it loads links in the right language and adds the correct `dir` value too.

**Describe alternatives you've considered**

**Testing notes**
Regression of CopyRight Container story. 

Dev insight: Cypress test for Storybook may need to change, if so pair with test. 

**Additional context**
We shouldn't need stories for all of our containers (e.g. Blocks, or Fragments) since these are for handling our Optimo data.

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2195

**Overall change:** _Add CopyRight container to storybook._

**Code changes:**

- _add copyright container story_
- _using knobs to have a drop down of existing services_ 

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/containers/Copyright/index.stories.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { withKnobs } from '@storybook/addon-knobs';
+import CopyRightContainer from '.';
```
Comment (12) [inline src/app/containers/Copyright/index.stories.jsx]:
```suggestion
import CopyrightContainer from '.';
```

File: src/app/containers/Copyright/index.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import services from '../../lib/config/services';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+
+storiesOf('CopyRight Container', module)
```
Comment (12) [inline src/app/containers/Copyright/index.stories.jsx]:
```suggestion
storiesOf('Copyright Container', module)
```

File: src/app/containers/Copyright/index.stories.jsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+          services[service].imageCopyrightOffscreenText || 'Image source, ';
+
+        const serviceContextStub = {
+          imageCaptionOffscreenText,
+          lang: services[service].lang,
+          dir: services[service].dir,
+        };
+        return (
+          <ServiceContextProvider service={service} value={serviceContextStub}>
+            <CopyRightContainer>{imageCaptionOffscreenText}</CopyRightContainer>
```
Comment (12) [inline src/app/containers/Copyright/index.stories.jsx]:
```suggestion
            <CopyrightContainer>{imageCaptionOffscreenText}</CopyrightContainer>
```

Also is `imageCaptionOffscreenText` the right name here? The story is showing the offscreen text as the copyright text, so it's not quite making sense. `imageCaptionText` would suffice.

**Review by ryanmccombe — APPROVED** [review]

File: src/app/containers/Copyright/index.stories.jsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        const imageCaptionText =
+          services[service].imageCopyrightOffscreenText || 'Image source, ';
+
+        const serviceContextStub = {
+          imageCaptionOffscreenText: imageCaptionText,
+          lang: services[service].lang,
+          dir: services[service].dir,
+        };
+        return (
+          <ServiceContextProvider service={service} value={serviceContextStub}>
```
Comment (dr3) [inline src/app/containers/Copyright/index.stories.jsx]: ServiceContextProvider doesnt take `value`, do you mean `ServiceContext.Provider`? if so, it doesnt take service

Comment (thekp) [reply src/app/containers/Copyright/index.stories.jsx]: i see, updating now to use ServiceContext.Provider as the container does not require service prop anyways

**Review by dr3 — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing, it's just adding a story

**Pull request merged**